flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceBoundedGenericRecord.java

```diff
@@ -69,12 +70,13 @@ public class TestIcebergSourceBoundedGenericRecord {
   private static final HadoopCatalogExtension CATALOG_EXTENSION =
       new HadoopCatalogExtension(TestFixtures.DATABASE, TestFixtures.TABLE);
 
-  @Parameters(name = "format={0}, parallelism = {1}")
+  @Parameters(name = "format={0}, parallelism = {1}, useConverter = {2}")
   public static Object[][] parameters() {
     return new Object[][] {
-      {FileFormat.AVRO, 2},
-      {FileFormat.PARQUET, 2},
-      {FileFormat.ORC, 2}
+      {FileFormat.AVRO, 2, false},
+      {FileFormat.PARQUET, 2, true},
+      {FileFormat.PARQUET, 2, false},
+      {FileFormat.ORC, 2, true}
```

Comment:
Why did you opt for this exact parametrization?

I happy for having tests for Parquet in both cases, as that is the main usecase for us now:
```
      {FileFormat.PARQUET, 2, true},
      {FileFormat.PARQUET, 2, false},
```

I might have opted for testing the future features, and keep one test for backward comp, like this:
```
      {FileFormat.AVRO, 2, true},
      {FileFormat.PARQUET, 2, true},
      {FileFormat.PARQUET, 2, false},
      {FileFormat.ORC, 2, true}
```

Reply:
sounds good to me. will change